src/WebJobs.Script/Extensions/FileUtility.cs

```diff
@@ -75,6 +75,9 @@ public static async Task WriteAsync(string path, string contents, Encoding encod
                 throw new ArgumentNullException(nameof(contents));
             }
 
+            var directory = Path.GetDirectoryName(path);
```

Comment:
I'll remove this and instead set the tests up to create the requisite directories